.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ jobs:
 
   e2e-testit:
     docker:
-      - image: solita/harja-cypress:latest
+      - image: solita/harja-cypress:testi-j11 # Vaihdettu väliaikaisesti 26.1.2022, voi vaihtaa takaisin :latest-tagiin.
```

Comment:
Order of operations:
Kun PR hyväksytään, pusketaan docker-image latest-tagilla hubiin ja vaihdetaan tämä takaisin sitä ennen mergeä.